core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy

```diff
@@ -97,7 +97,7 @@ l.layout(norefresh:true, permission:app.ADMINISTER, title:my.displayName, csscla
 
             f.bottomButtonBar {
                 f.submit(value:_("Save"))
-                f.apply()
+                f.apply(value:_("Apply"))
```

Comment:
The default value should be localized in the form. Otherwise other buttons won't be modified. https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/form/apply.jelly#L41